Packs/CloudConvert/Integrations/CloudConvert/CloudConvert.py

```diff
@@ -0,0 +1,396 @@
+    """
+    results = client.check_status(arguments)
+    results_data = results.get('data')
+
+    # Check if no 'data' field was returned from the request, meaning the input was invalid
+    if results_data is None:
+        if results.get('message'):
+            raise ValueError(results.get('message'))
+        else:
+            raise ValueError('No response from server, the server could be temporary unavailable or it is handling too '
+                             'many requests. Please try again later.')
+
+    # If checking on an export to entry operation, manually change the operation name
+    # For other operations, the operation matches the operation field in the API's response, so no change is needed
+    if argToBoolean(arguments.get('create_war_room_entry', False)) \
+            and results.get('data', {}).get('operation') == 'export/url':
+        results['data']['operation'] = 'export/entry'
+
+    # Check if an export to war room entry operation is finished
+    # If it did - create the entry
+    if results.get('data', {}).get('status') == 'finished' \
+            and argToBoolean(arguments.get('create_war_room_entry', 'False'))\
+            and results.get('data', {}).get('operation') == 'export/entry':
+        results_info = results.get('data', {}).get('result', {}).get('files', [{}])[0]
+        url = results_info.get('url')
+        file_name = results_info.get('filename')
+        file_data = client.get_file_from_url(url)
+        war_room_file = fileResult(filename=file_name, data=file_data)
+        readable_output = tableToMarkdown('Check Status Results', remove_empty_elements(results_data),
+                                          headers=('created_at', 'depends_on_task_ids', 'id', 'operation', 'result',
+                                                   'status'), headerTransform=string_to_table_header)
+        return_results(CommandResults(
+            outputs_prefix='CloudConvert.Task',
+            outputs_key_field='id',
+            raw_response=results,
+            readable_output=readable_output,
+            outputs=remove_empty_elements(results_data)
+        ))
+        return war_room_file
+
+    else:
+        readable_output = tableToMarkdown('Check Status Results', remove_empty_elements(results_data),
+                                          headers=('created_at', 'depends_on_task_ids', 'id', 'operation', 'result',
+                                                   'status'),
+                                          headerTransform=string_to_table_header)
+        return CommandResults(
+            readable_output=readable_output,
+            outputs_prefix='CloudConvert.Task',
+            outputs_key_field='id',
+            raw_response=results,
+            outputs=remove_empty_elements(results_data)
+        )
+
+
+@logger
+def export_command(client: Client, arguments: Dict[str, Any]):
```

Comment:
arguments should be explicitly stated for all the command functions.
in the main function, you can unpack them using `**demisto.args()`